Src/BootCamp.Chapter/TextBoxPrinter.cs

```diff
@@ -1,12 +1,137 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
     public static class TextBoxPrinter
     {
-        // TODO: Implement.
-        public static string Print(object obj) => "";
+        public static string Print(object obj)
+        {
+            var atri = AttributesGetter.GetClassAttribute<TextableAttribute>(obj.GetType());
+            if (atri != null)
+            {
+                return MakePadding(obj.ToString(), atri.Padding, atri.Horizontal, atri.Vertical, atri.Corner);
+            }
+            else
+            {
+                return obj.ToString();
+            }
+        }
+
+        public static string MakePadding(string message, int padding = 0, char horizontal = '-', char vertical = '|', char corner = '+')
```

Comment:
Make is a bit of an unconventional name. BuildPadding is a bit better.
I think an even better name would be to just ``Print``. The context (``TextBoxPrinter``) implies on the padding.